### PR TITLE
Rm birthday banner

### PR DESCRIPTION
**Pull request by shanioffe**
Removes the CLARK 5th birthday banner from homepage.

### CODE REVIEW

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss [hunk 1]
```diff
@@ -34,6 +34,7 @@ h1 {
   //Wrapper Outer Parent
   display: inline-block;
   margin-top: 1%;
+  max-width: 300px;
```
Comment (mfranz1) [inline src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss:37]: Just curious, why was this added?

**Review by mfranz1 — CHANGES_REQUESTED** [review]
There's git module file. That shouldn't be apart of this PR

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss (hunk 1 above)
Comment (shanioffe) [reply src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss:37]: I'm not sure how those got there, I can remove them.

**Review by mfranz1 — APPROVED** [review]

**Review by mfranz1 — CHANGES_REQUESTED** [review]
`src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss` Should not have been deleted. Just the max-width.

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss (hunk 1 above)
Comment (mfranz1) [reply src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss]: Just the max-width I think should have been deleted, not the entire file.

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss [hunk 2]
```diff
@@ -1,116 +0,0 @@
-@import 'vars.scss';
```
Comment (JosueMolinaMorales) [inline src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss:1]: Why was this deleted?

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss (hunk 1 above)
Comment (JosueMolinaMorales) [reply src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss]: yeah the entire should not have been deleted.

Comment (shanioffe) [reply src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss]: Oh no, sorry guys :O

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss (hunk 2 above)
Comment (pzalep1) [reply src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss:1]: ^

**Review by pzalep1 — COMMENTED** [review]
Just the one question from Josue.

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss (hunk 2 above)
Comment (shanioffe) [reply src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss]: I had the wrong understanding of how github worked. I thought that this was an extraneous file that made it into my PR by mistake and didn't realize that deleting it from my PR on github would also delete it from the clark-client. I'm fixing it now

**Review by Cwagne17 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

File: src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss (hunk 1 above)
Comment (JosueMolinaMorales) [reply src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss]: All good! Mistakes happen. This is why we have this review process :)

**Review by mfranz1 — APPROVED** [review]

**Review by pmanal1 — APPROVED** [review]
